RockWeb/Blocks/Finance/TransactionEntry.ascx.cs

```diff
@@ -2653,9 +2777,18 @@ private void SaveTransaction( FinancialGateway financialGateway, GatewayComponen
             batch.ControlAmount = newControlAmount;
 
             transaction.BatchId = batch.Id;
+            transaction.LoadAttributes( rockContext );
```

Comment:
it might be better to have a Block Attribute [AttributeField("Allowed Transaction Attributes From URL", "Specify any Transaction Attributes that can be populated from the URL", category: "Advanced"....)]